content/en/docs/Tasks/make-catalog-available-on-cluster.md

```diff
@@ -85,4 +85,23 @@ spec:
       interval: 10m
 ```
 
-[creating-a-catalog-steps]: /docs/tasks/creating-a-catalog/#creating-a-catalog
\ No newline at end of file
+If the `imagePullSecret` is referenced in the bundle, for instance when the controller-manager image is pulled from a private registry, there is no place in the API to tell OLM to attach the `imagePullSecrets`. As a consequence, permissions to pull the image should be added directly to the controller-manager deployment configurations. This can be achieved by updating `config/manager/manager.yaml` and adding `deployment.spec.template.spec.imagePullSecrets` with the required secret name. 
```

Comment:
Can you make this comment a little bit more generic? Instead of `added directly to the controller-manager deployment configurations`, something like `added directly to the operator deployment's podspec`. Then update that last sentence to contextualize it around "here's how that works for the operator-sdk's abstraction" in some way?